### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by sviatoslav-krupa**
**CHECK LIST**
- [ ] specs added
- [x] specs passed
- [x] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [ ] I've checked new feature as logged in and logged out user if needed
- [x] "let!" is used in specs
- [ ] screenshot of rails_best_practices and other tools added
- [x] screenshot of feature is added if task has UI

1. Add small changes to partials (wrap it on container)
2. Add new page type (BY_AJAX) and update CompanyPagesAnalytics class
3. Update reek for max number of constants (15) for previous item
4. Update icons on dashboard

icons

### CODE REVIEW

File: app/views/account/my_tenants/edit.html.slim [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
-              'Edit my tenant' => nil})
-      div
-        .row
-          .col-md-3.col-md-offset-3
-            = simple_form_for @tenant, url: account_my_tenant_path do |f|
-              = f.input :name
-              = f.input :website
-              = f.input :phone
-              = f.button :submit, 'Save Tenant', class: 'btn btn-primary'
+.content-wrapper
```
Comment (romaslmd) [inline app/views/account/my_tenants/edit.html.slim]: invalid bootstrap grid
wrap `.row` in either`.container` or `.container-fluid`

File: app/lib/company_pages_analytics.rb [hunk 2]
```diff
@@ -1,8 +1,9 @@
 class CompanyPagesAnalytics
+  MANUALLY_ADDED_PAGES_TYPES = [Page::CHROME_EXTENSION, Page::BY_AJAX].freeze
   def self.count_pages(current_tenant)
     current_tenant.pages.where(
-      'page_type != ? and pages.created_at > ?',
-      Page::CHROME_EXTENSION, 1.day.ago
+      'page_type not in (?) and pages.created_at > ?',
```
Comment (igorkasyanchuk) [inline app/lib/company_pages_analytics.rb]: NOT IN is used for arrays
find another SQL operator for "!="

File: app/lib/company_pages_analytics.rb [hunk 3]
```diff
@@ -1,8 +1,9 @@
 class CompanyPagesAnalytics
+  MANUALLY_ADDED_PAGES_TYPES = [Page::CHROME_EXTENSION, Page::BY_AJAX].freeze
```
Comment (igorkasyanchuk) [inline app/lib/company_pages_analytics.rb]: move to Page

File: app/models/page.rb [hunk 4]
```diff
@@ -30,11 +30,13 @@ class Page < ApplicationRecord
   LINKEDIN_TYPE    = 'linkedin'.freeze
   CRUNCHBASE_TYPE  = 'crunchbase'.freeze
   CHROME_EXTENSION = 'chrome_extension'.freeze
+  BY_AJAX = 'by_ajax'.freeze
```
Comment (igorkasyanchuk) [inline app/models/page.rb]: format "=" (add spaces)

File: app/lib/company_pages_analytics.rb (hunk 2 above)
Comment (sviatoslav-krupa) [reply app/lib/company_pages_analytics.rb]: but I have array here
image

Comment (igorkasyanchuk) [reply app/lib/company_pages_analytics.rb]: yes. now yes

Comment (sviatoslav-krupa) [reply app/lib/company_pages_analytics.rb]: so, it's good or I must change it?

**Pull request merged**